openapi.yaml

```
@@ -243,6 +233,7 @@ paths:
         - OI
       summary: Get all notes of a trouble ticket
       description: Get all notes of a trouble ticket
+      operationId: getMalFaconNotes
```

pour garder le meme type de nommage que sur le reste on a pas mis de F maj dans Malfacon

```suggestion
      operationId: getMalfaconNotes
```